sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/JsonWebKey.java

```diff
@@ -1038,22 +932,11 @@ public boolean hasPrivateKey() {
      *
      * @return true if the {@link JsonWebKey} is valid; false otherwise.
      */
-    @JsonIgnore
     public boolean isValid() {
         if (keyType == null) {
             return false;
         }
 
-        if (keyOps != null) {
-            final Set<KeyOperation> set = new HashSet<>(KeyOperation.values());
-
-            for (KeyOperation keyOp : keyOps) {
-                if (!set.contains(keyOp)) {
-                    return false;
-                }
-            }
-        }
-
```

Comment:
The issue is this never worked. Since we're validation a list of `KeyOperation`s the user either used the constant or had to use `fromString` to create them. In both cases this will result in that string value being in `KeyOperation.values()`, resulting in this check doing nothing.

Reply:
I see, so creating a new `KeyOperation` using `fromString()` adds the new entry to the static collection of known values?